### DISCUSSION TITLE
Images in PDF/Email Templates - related record

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#14275] [Feature improvement]
Just read below about images - anyone have cracked to show images in related list in Email Template or PDF?

**Comment by mariuszkrzaczkowski** [comment]
give an example

**Comment by JohnMaverick** [comment]
For instance, I have custom module and related to Accounts module. Now I need to print report including all related Accounts together with one or more picture field(s) (for instance their logo). Currently pasting variable of custom image fields does nothing.
For me it looks like quite important to be able to make reports including image fields.

**Reply by TonyM15** [reply]
aw, that is pretty essential. Thumb up here
